hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/AcceptanceTestProperties.java

```diff
@@ -54,8 +54,7 @@ public class AcceptanceTestProperties {
     @Max(5)
     private int maxRetries = 2;
 
-    @NotNull
-    private Long maxTinyBarTransactionFee = 2_000_000_000L;
+    private long maxTinyBarTransactionFee = 40_00_000_000L;
```

Comment:
Would be a bit easier to read if it was `4_000_000_000L`

Reply:
not really imo. since 1 hbar = 10_000_000 th, so it's actually easier this way since then you immediately know it's 40 hbars.

Reply:
Perhaps instead convert currency units in file using `Hbar`:

```suggestion
    private long maxTinyBarTransactionFee = Hbar.from(40).toTinyBars();
```

Reply:
In the future, we can consider registering a custom Spring converter so we can specify `40h` or `40th`.

Reply:
sure